src/main/java/com/synopsys/integration/alert/web/config/JobConfigActions.java

```diff
@@ -220,47 +219,18 @@ public String testJob(final JobFieldModel jobFieldModel, final String destinatio
             final Optional<TestAction> testActionOptional = descriptorProcessor.retrieveTestAction(channelFieldModel);
             if (testActionOptional.isPresent()) {
                 final Map<String, ConfigurationFieldModel> fields = createFieldsMap(channelFieldModel, otherJobModels);
-
+                // TODO the custom message fields are not written to the database or defined fields in the database.  Need to manually add them.
```

Comment:
I think the TODO should be on the second comment, that way the first one does not get deleted when the TODO is resolved. Was the intention to resolve this TODO in the PR?

Reply:
I will move the TODO to the second line.  The intention is not to solve it in this PR but in a future enhancement as tech debt.